### ISSUE TITLE
Linked pages don't have the JS injected

### ISSUE THREAD

**Issue by elordin** [links-lang/links#161]
```
Links-version: 0.6.1 @ 568ff46
```

```
[ ] Proposal
[x] Bug
[ ] Feature Request
[ ] Other
```

Minimal example:
```
fun makePage () {
  page
    <html>
      <body><a l:href="{ makePage() }">click</a></body>
    </html>
}

fun main () {
  addStaticRoute("/", "src/static", [ ("js", "application/javascript") ]);
  addRoute("/test", fun (_, _) { makePage() });

  servePages()
}

main()
```

When visiting the page for the first time / without `GET` parameter the source code is:
```
<html>
<head>
  <script type='text/javascript' src="http://localhost:8080/lib/regex.js"></script>
    <script type='text/javascript' src="http://localhost:8080/lib/yahoo/yahoo.js"></script>
    <script type='text/javascript' src="http://localhost:8080/lib/yahoo/event.js"></script>
    <script type='text/javascript'> /* links JS code */</script>
  </head>
  <body>
    <script type='text/javascript'> /* links JS code */</script>
    <a href="?_k=BOUHAQIBAzk5NgTfFAECAQQyNjU0BN4UAA==">click</a>
  </body>
</html>
```

but as soon as one clicks that link, the source becomes
```
<html>
  <body>
    <a href="?_k=BOUHAQIBAzk5NgTfFAECAQQyNjU0BN4UAA==">click</a>
  </body>
</html>
```

**Comment by SimonJF** [comment]
I think this is a duplicate of #139.

**Comment by elordin** [comment]
It very much is
Closed as duplucate of #139
